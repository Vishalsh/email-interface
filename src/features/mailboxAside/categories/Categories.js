import React from 'react';

import classes from './Categories.module.scss';

const categories = [{
  name: 'Work',
  color: '#1bb394'
}, {
  name: 'Documents',
  color: '#ef5352'
}, {
  name: 'Social',
  color: '#1c84c6'
}, {
  name: 'Advertising',
  color: '#23c6c8'
}, {
  name: 'clients',
  color: '#f8ac59'
}];

const Categories = () => {
  return (
    <ul className={classes.list}>
      {
        categories.map((category) => (
          <li key={category.name} className={classes.listItem}>
            <i className={classes.icon} style={{'background': category.color}} />
            <span>{category.name}</span>
          </li>
        ))
      }
    </ul>
  )
};

export default Categories;
