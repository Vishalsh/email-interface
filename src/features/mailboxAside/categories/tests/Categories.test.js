import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Categories from '../Categories';

describe('Categories', () => {
  const component = shallow(<Categories />);

  it('should render the component', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});