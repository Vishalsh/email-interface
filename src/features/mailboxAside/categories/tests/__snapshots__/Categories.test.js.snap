// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Categories should render the component 1`] = `
<ul
  className="list"
>
  <li
    className="listItem"
    key="work"
  >
    <i
      className="icon work"
    />
    <span
      className="name"
    >
      work
    </span>
  </li>
  <li
    className="listItem"
    key="documents"
  >
    <i
      className="icon documents"
    />
    <span
      className="name"
    >
      documents
    </span>
  </li>
  <li
    className="listItem"
    key="social"
  >
    <i
      className="icon social"
    />
    <span
      className="name"
    >
      social
    </span>
  </li>
  <li
    className="listItem"
    key="advertising"
  >
    <i
      className="icon advertising"
    />
    <span
      className="name"
    >
      advertising
    </span>
  </li>
  <li
    className="listItem"
    key="clients"
  >
    <i
      className="icon clients"
    />
    <span
      className="name"
    >
      clients
    </span>
  </li>
</ul>
`;
